### PR TITLE
Add nested LS_CE_AOV CPU period-finding algorithm

### PR DESCRIPTION
**Pull request by bfhealy**
This PR adds a nested `LS_CE_AOV` CPU period-finding algorithm to scope. This can be used by `generate_features.py` in the same way as the GPU-accelerated `ELS_ECE_EAOV` algorithm.

### CODE REVIEW

File: tools/generate_features.py [hunk 1]
```diff
@@ -1008,7 +1029,7 @@ def generate_features(
 
         print(f'Computing Fourier stats for {len(period_dict)} algorithms...')
         for algorithm in period_algorithms:
-            if algorithm != 'ELS_ECE_EAOV':
+            if (algorithm != 'ELS_ECE_EAOV') & (algorithm != 'LS_CE_AOV'):
```
Comment (mcoughlin) [inline tools/generate_features.py:1032]: if algorithm not in ["ELS_ECE_EAOV", "LS_CE_AOV"]:

File: tools/generate_features.py [hunk 2]
```diff
@@ -987,7 +1008,7 @@ def generate_features(
             pdot_dict['Ones'] = np.ones(len(tme_collection))
 
         for algorithm in period_algorithms:
-            if algorithm != 'ELS_ECE_EAOV':
+            if (algorithm != 'ELS_ECE_EAOV') & (algorithm != 'LS_CE_AOV'):
```
Comment (mcoughlin) [inline tools/generate_features.py:1011]: if algorithm not in ["ELS_ECE_EAOV", "LS_CE_AOV"]:

**Review by mcoughlin — APPROVED** [review]
small suggestion

File: tools/generate_features.py (hunk 1 above)
Comment (bfhealy) [reply tools/generate_features.py:1032]: This is better syntax, thanks!

**Pull request merged**